### PR TITLE
translate peft.md to chinese

### PR DESCRIPTION
**Pull request by jiaqiw09**
# What does this PR do?

Part of  #26803 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests? _not necessary_

## Who can review?
@stevhliu

### CODE REVIEW

**Comment by jiaqiw09** [comment]
@stevhliu  

hi, here is another pr for peft.md translation.

And I just check the  _toctree.yml file, and I think I will first finish section `Tutorials` and `Developer guides`. I think it may be helpful for people who want to try `transformers`. Then I will continue to work on anther sections. When we finsihed all translation work with other contributors, I think it's better to look back and update some link info, small typos or wrong translation words.

Just try to do some work by the end of this year as many people will try to use transformers as it's the trend for DL work.

Best

File: docs/source/zh/peft.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+[参数高效微调（PEFT）方法](https://huggingface.co/blog/peft)在微调过程中冻结预训练模型的参数，并在其顶部添加少量可训练参数（adapters）。adapters被训练以学习特定任务的信息。这种方法已被证明非常节省内存，同时具有较低的计算使用量，同时产生与完全微调模型相当的结果。
+
+使用PEFT训练的adapters通常比完整模型小一个数量级，使其方便共享、存储和加载。
+
+<div class="flex flex-col justify-center">
+  <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/peft/PEFT-hub-screenshot.png"/>
+  <figcaption class="text-center">与完整尺寸的模型权重（约为700MB）相比，存储在Hub上的OPTForCausalLM模型的adapter权重仅为~6MB。</figcaption>
+</div>
+
+<br>
```
Comment (stevhliu) [inline docs/source/zh/peft.md:25]: I don't think we need this

```suggestion
```

File: docs/source/zh/peft.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+model_id = "facebook/opt-350m"
+peft_model_id = "ybelkada/opt-350m-lora"
+
+model = AutoModelForCausalLM.from_pretrained(model_id)
+model.load_adapter(peft_model_id)
+```
+
+## 基于8bit或4bit进行加载
+
+`bitsandbytes`集成支持8bit和4bit精度数据类型，这对于加载大模型非常有用，因为它可以节省内存（请参阅`bitsandbytes`集成指南以了解更多信息）。要有效地将模型分配到您的硬件，请在[`~PreTrainedModel.from_pretrained`]中添加`load_in_8bit`或`load_in_4bit`参数，并将`device_map="auto"`设置为：
```
Comment (stevhliu) [inline docs/source/zh/peft.md:88]: I think we're missing a link to the bitsandbytes integration guide somewhere here

**Review by stevhliu — COMMENTED** [review]
Thanks for another translation!

I fully agree with your plan, and feel free to continue to ping me for any reviews or if I can help you with anything else 🤗

**Comment by jiaqiw09** [comment]
@stevhliu 

Hi, I have fixed problems.

And for `<br>`, I just think it is too compact to read easily. But I still delete this line.

Best

**Review by stevhliu — APPROVED** [review]
Thanks for the translation!

**Pull request merged**